lib/rules/metadata/profile.js

```diff
@@ -145,6 +145,14 @@ exports.check = async function (sr, done) {
             sr.throw(
                 `[EXCEPTION] The document "${docTitle}" seems to be an Editor's Draft, which is not supported.`
             );
+        } else if (
+            sr.jsDocument.querySelector(
+                'link[href^="https://www.w3.org/StyleSheets/TR/"]'
+            )
+        ) {
+            sr.throw(
+                `Pubrules is having a hard time identifying the profile of the document "${docTitle}" from its &lt;h2&gt;. Please make sure the &lt;h2&gt; is in a valid format: <pre><code>&lt;h2&gt;W3C (type of the document) DD Month YYYY&lt;/h2&gt;</code></pre>`
```

Comment:
Please list all the profiles doc and point to `#dateTitleH2` so people will know exactly the h2 needed.